README.md

```diff
@@ -249,90 +249,129 @@ For the out-of-the-box implementation of capturing exceptions thrown out of your
 
 To capture exceptions that occur within the framework presentation layer (or any other area that is handling exceptions), refer to that frameworks documentation about handling exceptions, and send the exception to Raygun using the techniques described above (the static accessor will help out here)
 
 ## Play 2 Framework for Java and Scala
 
-This provider now contains a dedicated Play 2 provider for automatically sending Java and Scala exceptions from Play 2 web apps. Feedback is appreciated if you use this provider in a Play 2 app. You can use the plain core-3.x.x provider from Scala, but if you use this dedicated Play 2 provider HTTP request data is transmitted too.
+This provider now contains a dedicated Play 2 provider for automatically sending Java and Scala exceptions from Play 2 web apps. Feedback is appreciated if you use this provider in a Play 2 app. You can use the plain core-4.x.x provider from Scala, but if you use this dedicated Play 2 provider, HTTP request data is transmitted too.
 
 ### Installation
 
 #### With SBT
 
-Add the following line to your build.sbt's libraryDependencies:
+Add the following line to your `build.sbt`'s `libraryDependencies`:
 
 ```
 libraryDependencies ++= Seq(
-    "com.mindscapehq" % "raygun4java-play2" % "3.0.0"
+    "com.mindscapehq" % "raygun4java-play2" % "4.0.0"
 )
 ```
 
 #### With Maven
 
-Add the raygun4java-play2-3.x.x dependency to your pom.xml (following the instructions under 'With Maven and a command shell' at the top of this file).
+Add the `raygun4java-play2-4.x.x` dependency to your `pom.xml` (following the instructions under 'With Maven and a command shell' at the top of this file).
 
 ### Usage
 
-For automatic exception sending, in your Play 2 app's global error handler, RaygunPlayClient has a method which allows you to pass in a RequestHeader and send a Throwable:
+To handle exceptions automatically, define a custom error handler by creating a class that inherits from `HttpErrorHandler`. For Play 2's newer versions, `GlobalSettings.onError` is no longer available. Instead, you should use `HttpErrorHandler`.
 
-**In Scala**
-
-**app/Global.scala**
-```scala
-override def onError(request: RequestHeader, ex: Throwable) = {
-  val rg = new RaygunPlayClient("your_api_key", request)
-  val result = rg.Send(ex)
-
-  super.onError(request, ex)
-}
+1. Add the following line to your `conf/application.conf`:
+```
+play.http.errorHandler = "com.raygun.CustomErrorHandler"
 ```
 
-**In Java**
+2. Create the custom error handler class:
+
+**In Java:**
 
-**app/Global.java**
 ```java
+package com.raygun;
```

Comment:
I think this provider still uses the MindscapeHQ namespace, right? And I am not keen to put everyone through what we just had to do with javax to jakarta change..

Reply:
Ok, I am seeing now you are declaring an example package but the import afterwards is ok. Ignore this (just had my first sip of coffee)